internal/project/service.go

```diff
@@ -163,3 +165,23 @@ func (s *ServiceImpl) GetAllProjects(ctx context.Context) (*[]Project, error) {
 
 	return projects, nil
 }
+
+func (s *ServiceImpl) GetProjectByID(ctx context.Context, id int) (*Project, error) {
+	project, err := s.repo.GetProjectByID(ctx, id)
+
+	if err != nil {
```

Comment:
We usually don't add a blank line before the `err != nil` check in order to keep it semantically closer to its context